app/services/order_details/param_updater.rb

```diff
@@ -90,6 +90,7 @@ def assign_price_changed_by_user
       @order_detail.price_changed_by_user = nil
     elsif %w[actual_cost actual_subsidy price_change_reason].any? { |a| @order_detail.changed.include?(a) }
       @order_detail.price_changed_by_user = @editing_user
+      LogEvent.log(@order_detail, :price_change, @editing_user)
```

Comment:
I'm about 95% sure it will get rolled back, but can we add a spec where we hit this code path, but something else causes the order detail to be invalid and not save. We should make sure this doesn't get created in the situation where the update doesn't happen.

Reply:
Sure I'll add it